src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {

```

Comment:
```suggestion
public class Ball {
```
those are absolutely different objects